### DISCUSSION TITLE
Add Instance Zones for PQ

### DISCUSSION THREAD

**Discussion by JDS300** [smasherprog/EqTool#87] [Ideas]
Can the instance zones (Sol B, Perma, Sky, Hate,  Fear, etc.) be added to the map? They do not appear to get detected correctly likely due to the name being different.

**Comment by smasherprog** [comment]
Yes, I need to know the zone names for those instances so I can add them
For example

You have zoned into The plane of hate

**Comment by smasherprog** [comment]
I dont play on PQ currently so i need others to do this leg work

**Reply by JDS300** [reply]
[Fri Jan 05 17:23:04 2024] You have entered Nagafen's Lair (Instance).

I believe they are the same zone names with just (Instance) at the end.

**Comment by smasherprog** [comment]
Perfect ill be able to add this tonight thanks

**Comment by smasherprog** [comment]
Just added this code it will be heading out   soon. I have other things im adding to this release as well

**Reply by JDS300** [reply]
Awesome, thank you!

**Comment by JDS300** [comment]
This still doesn't change the map, but that might be because the zone names are also different.  Looking at https://www.pqdi.cc/zones it looks like the zone name is zonename_instanced, for example Fear is 'fear_instanced = Plane of Fear (Instanced)'.

Otherwise, it's 'fearplane = Plane of Fear' for the non-instanced version.

**Comment by smasherprog** [comment]
I had 

"plane of fear (Instance)"

Updated to 

"plane of fear (Instanced)"

Case doesnt matter above, but looks like i was missing the d   ?

code pushed will be available in 5 minutes

**Reply by JDS300** [reply]
Cool, will see how it picks up the zones now.  Looking at the zone list it looks like some are 'Instance' and some are 'Instanced'. So it's not the same for each zone. 

Thank you!

**Comment by smasherprog** [comment]
Fixed the names from the link you gave me. It wasnt working earlier...
